README.md

```diff
@@ -1,2 +1,230 @@
-# terraref-datasets
-Repository for code and small datasets derived from the TERRA REF program
+# [TERRA-REF Data](https://www.terraref.org/)
+
+This repository contains source code for accessing and curating [TERRA-REF](https://terraref.ncsa.illinois.edu/bety/) data used to support the [GenoPhenoEnvo](https://github.com/genophenoenvo) project and machine learning research. This repository supports one of our goals to provide open data and reproducible code in order to follow [FAIR](https://www.go-fair.org/fair-principles/) data principles and contribute to open science.
+
+This repository focuses on _Sorghum bicolor_ trait data collected from four experiments and the associated weather data for those locations, listed below.
+
+- Maricopa Agricultural Center, University of Arizona, Season 4
+    - Coordinates: 33.069, -111.972
+    - Elevation: 362 meters
+    - Planting: 2017-04-20, Day 110
+    - Last Day of Harvest: 2017-09-16, Day 259
+- Maricopa Agricultural Center, University of Arizona, Season 6
+    - Coordinates: 33.068941, -111.972244
+    - Elevation: 362 meters
+    - Planting: 2018-04-25, Day 115
+    - Harvest: 2018-08-01, Day 213
+- Kansas State University, Ashland Bottoms
+    - Coordinates: 39.126, -96.677
+    - Elevation: 325 meters
+    - Planting: 2016-06-17, Day 169
+    - Harvest: 2016-10-21, Day 295
+- Clemson University Pee Dee Research and Education Center, South Carolina
+    - Coordinates: 34.289, -79.737 
+    - Elevation: 42 meters
+    - Planting: 2014-05-06, Day 126
+    - Latest date in Clemson trait data: 2014-10-15, Day 288
+    
+### Licenses
+TERRA-REF data and other documents are licensed under CC-By. All software is licensed under MIT.
+
+### Contacts
+Please contact these individuals with any questions
+
+- Todd Mockler, Project/Genomics Lead (email: tmockler@danforthcenter.org)
+- David LeBauer, Computing Pipeline Lead (email: dlebauer@email.arizona.edu)
+- Nadia Shakoor, Project Director (email: nshakoor@danforthcenter.org)
+
+### Citations
+
+#### Dryad Data 
+
+Maricopa Agricultural Center (MAC) data from the TERRA-REF project seasons four and six have been published with Dryad and can be downloaded [here](https://doi.org/10.5061/dryad.4b8gtht99).
+
+**LeBauer, David et al. (2020), Data From: TERRA-REF, An open reference data set from high resolution genomics, phenomics, and imaging sensors, v6, Dryad, Dataset, https://doi.org/10.5061/dryad.4b8gtht99**
+
+---
```

Comment:
Data from KSU are unpublished.

Data from Clemson come from a separate citation that should be listed here